app/main.py

```diff
@@ -41,3 +46,26 @@ def make_change(transaction_id : int, transaction: Transaction):
         return {"Error": "Transaction already exists"}
     transactions[transaction_id] = transaction
     return calculate_change(transactions[transaction_id].currency_inserted, transactions[transaction_id].price,  [25, 10, 5, 1])
+
+@app.put("/update-transaction/{transaction_id}")
+def update_transaction(transaction_id: int, transaction: UpdateTransaction):
+    if transaction_id not in transactions:
+        return {"Error": "Transaction does not exist"}
+
+    if transaction.item_name != None:
+        transactions[transaction_id]["item_name"] = transaction.item_name
+
+    if transaction.price != None:
+        transactions[transaction_id]["price"] = transaction.price
+
+    if transaction.currency_insterted != None:
+        transactions[transaction_id]["currency_inserted"] = transaction.currency_insterted
+
+    return transactions[transaction_id]
+
+@app.delete("/delete-transaction/{transaction_id}")
```

Comment:
lets create a post as well so we can create new ones

Reply:
I have a post route that should be showing on lines 38-43 on the code that is currently uploaded. I double checked the current main file and found it in there.

Reply:
Ah I missed that thank you